common/samples/ConsoleApp1/src/ConsoleApp1/Program.cs

```diff
@@ -1,7 +1,4 @@
 ﻿using System;
-using System.Collections.Generic;
```

Comment:
The common samples are snap shots of the Visual Studio project templates. We prefer that they don't diverge. If you want to propose changes to the project templates you can contributed to https://github.com/aspnet/templates.